src/Resources/translations/SonataPageBundle.en.xliff

```diff
@@ -358,8 +358,8 @@
         <source>sidemenu.link_list_snapshots</source>
         <target>Publications</target>
       </trans-unit>
-      <trans-unit id="90" resname="snapshot">
-        <source>snapshot</source>
+      <trans-unit id="90" resname="form.group_snapshot">
+        <source>form.group_snapshot</source>
         <target>Publications</target>
```

Comment:
Well It was a thing that confusing me, Here says publications but in the code uses `snapshot`, 

Don't you think we should use `Snapshots` instead of `Publications`?

Reply:
@VincentLanglet @jordisala1991

Reply:
But to translate for others languages I guess publication make more sense :/

I don't know what do you think?

Reply:
I think it should be done on another PR. What word does the user undestand more, publications or snapshots? I think snapshot is more technical word, but not sure, because I have never used this bundle.

Reply:
Yeah I guess it can be handled in other PR!